active-mq-artemis/src/main/scala/jms4s/activemq/activeMQ.scala

```diff
@@ -20,33 +20,29 @@ object activeMQ {
   case class Endpoint(host: String, port: Int)
   case class ClientId(value: String) extends AnyVal
 
-  def makeConnection[F[_]: ContextShift: Sync: Logger](
+  def makeContext[F[_]: ContextShift: Sync: Logger: Concurrent](
```

Comment:
`Concurrent` implies `Synch` IIRC..

Reply:
We should choose if propagate it to makeContext or make Concurrent in receiveJmMessage method

Reply:
Yep, for our `JmsContext` `Concurrent` is only needed for receiving since all other ops (low-level `send`s, etc..) will use a `Blocker`.
If we think about a typical usage of this lib, all our builders in `JmsClient` will need `Concurrent`, so I think we can keep this as-is?

Reply:
agreed